### ISSUE TITLE
[Question] Why can't I use Telegram with screen readers?

### ISSUE THREAD

**Issue by MesterPerfect** [telegramdesktop/tdesktop#16091]
Hello
I am a blind user
I am using windows8.1 operating system
I cannot use Telegram with screen readers
My screen reader cannot read anything inside the program
Are there solutions?
thanks

**Comment by ilya-fedin** [comment]
Duplicate of #476
